components/molecule/badgeCounter/src/styles/settings.scss

```
@@ -30,6 +30,10 @@ $molecule-badge-counter-large-offset-right: -($sz-base + $sz-base * 0.5) !defaul
 $fz-molecule-badge-counter-large: $fz-xs !default;
 $p-molecule-badge-counter-large: 0 $p-base !default;
 
+/* threeCharacters */
+$bdrs-molecule-badge-counter-threeCharacters: $bdrs-xxxl !default;
```

```suggestion
$bdrs-molecule-badge-counter-threeCharacters: $bdrs-molecule-badge-counter !default;
```
it has to be like this for retro-compatibility and later replace it to `$bdrs-xxxl` on your theme

---

But, `$bdrs-molecule-badge-counter` is `$bdrs-xxl` (16px) and for `$bdrs-molecule-badge-counter-threeCharacters` it's necessary `$bdrs-xxl` (32px), bu UX demand. For this reason, I have declared a new variable `$bdrs-molecule-badge-counter-threeCharacters`